<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrePRODIC3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_inicio">
      <property name="text">
       <string>Inicio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_malla">
      <property name="text">
       <string>Malla</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_variables">
      <property name="text">
       <string>Variables</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_valores">
      <property name="text">
       <string>Valores</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_bordes">
      <property name="text">
       <string>Bordes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_densidad">
      <property name="text">
       <string>Densidad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_salida">
      <property name="text">
       <string>Salida</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_load_data"/>
    <addaction name="action_save_data"/>
    <addaction name="action_generate_fortran_file"/>
    <addaction name="action_generate_and_view_results"/>
    <addaction name="action_generate_and_view_results_from_f90"/>
    <addaction name="action_view_results_from_tecplot"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_herramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="action_manual"/>
    <addaction name="action_settings"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_archivo"/>
   <addaction name="menu_herramientas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save_data">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar datos</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="text">
    <string>Manual de usuario</string>
   </property>
  </action>
  <action name="action_generate_fortran_file">
   <property name="text">
    <string>Generar archivo Fortran</string>
   </property>
  </action>
  <action name="action_generate_and_view_results">
   <property name="text">
    <string>Generar y visualizar resultados</string>
   </property>
  </action>
  <action name="action_load_data">
   <property name="text">
    <string>Cargar datos</string>
   </property>
  </action>
  <action name="action_generate_and_view_results_from_f90">
   <property name="text">
    <string>Generar y visualizar resultados a partir de archivo Fortran</string>
   </property>
  </action>
  <action name="action_view_results_from_tecplot">
   <property name="text">
    <string>Visualizar resultados a partir de archivo Tecplot</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Configuraci√≥n</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
